SPECS/azure-iot-sdk-c/azure-iot-sdk-c.spec

```diff
@@ -6,7 +6,7 @@ Name:           azure-iot-sdk-c
 # For apt-get packages they fix the version number to 0.2.0 and increase the release number with each release.
 # Since we want to control the release number as thr distribution, this scheme is not applicable for us.
 # They also used to use a regular versioning scheme like 1.3.7 but they did not tag their latest LTS with a version like that.
-Version:        2023.08.07
+Version:        2024.03.04
```

Comment:
You'd also need to update the `Source0` tag by adding the source URL (preferred) or the `# Source0` comment to point to the updated source. The URL must match with what we've got in `cgmanifest.json` and then the PR check should let you through.

Reply:
it was like that before, that said I'll update the spec

Reply:
note this will prevent auto-upgrade to work correctly